File: fox-hunt-domain/src/main/java/com/itechart/foxhunt/domain/entity/UserFeedbackEntity.java
package com.itechart.foxhunt.domain.entity;

import java.time.LocalDateTime;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;
import lombok.Data;

@Data
@Entity
@Table(name = "user_feedback", schema = "fh_admin")
public class UserFeedbackEntity {

  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  @Column(name = "user_feedback_id")
  private Long id;

  @Column(name = "ranking", nullable = false)
  private Integer ranking;

  @Column(name = "send_date", nullable = false)
  private LocalDateTime sendDate;

  @Column(name = "comment")
  private String comment;

  @Column(name = "has_read")
  private boolean hasRead;

  @ManyToOne
  @JoinColumn(name = "user_id",
      referencedColumnName = "app_user_id",
      nullable = false)
  private UserEntity user;

  @Column(name = "organization_id", nullable = false)
  private Long organizationId;

}

File: fox-hunt-organization-backend/src/common/entities/user-feedback.entity.ts
import { SCHEMA, USER_FEEDBACK_TABLE_NAME } from '../constants';
import {
  Column,
  Entity,
  JoinColumn,
  ManyToOne,
  PrimaryGeneratedColumn,
} from 'typeorm';
import { Transform } from 'class-transformer';
import { OrganizationEntity } from './organization.entity';

@Entity(USER_FEEDBACK_TABLE_NAME, { schema: SCHEMA })
export class UserFeedbackEntity {
  @Transform(({ value }) => +value)
  @PrimaryGeneratedColumn({ type: 'bigint', name: 'user_feedback_id' })
  userFeedbackId: number;

  @Column('character varying', { name: 'comment', nullable: true })
  comment: string | null;

  @Column('integer', { name: 'ranking', nullable: true })
  ranking: number | null;

  @Column('timestamp without time zone', { name: 'send_date' })
  sendDate: string;

  @Column('boolean', { name: 'has_read', default: () => 'false' })
  hasRead: boolean;

  @ManyToOne(
    () => OrganizationEntity,
    (organization) => organization.userFeedback,
    { eager: true },
  )
  @JoinColumn([{ name: 'organization_id', referencedColumnName: 'id' }])
  organizationEntity: OrganizationEntity;

  @Column({ type: 'bigint', name: 'user_id' })
  userId: number;
}
